src/content/guideline-metadata.ts

```diff
@@ -20,314 +23,525 @@ export interface GuidanceMetadata {
 //   AAA assessments)
-export const guidanceMetadataByAxeTag = {
-    wcag111: {
-        name: 'WCAG 1.1.1',
+const axeTagToGuidelineKeyMap = {
+    wcag111: 'WCAG 1.1.1',
+    wcag121: 'WCAG 1.2.1',
+    wcag122: 'WCAG 1.2.2',
+    wcag123: 'WCAG 1.2.3',
+    wcag124: 'WCAG 1.2.4',
+    wcag125: 'WCAG 1.2.5',
+    wcag131: 'WCAG 1.3.1',
+    wcag132: 'WCAG 1.3.2',
+    wcag133: 'WCAG 1.3.3',
+    wcag134: 'WCAG 1.3.4',
+    wcag135: 'WCAG 1.3.5',
+    wcag141: 'WCAG 1.4.1',
+    wcag142: 'WCAG 1.4.2',
+    wcag143: 'WCAG 1.4.3',
+    wcag144: 'WCAG 1.4.4',
+    wcag145: 'WCAG 1.4.5',
+    wcag1410: 'WCAG 1.4.10',
+    wcag1411: 'WCAG 1.4.11',
+    wcag1412: 'WCAG 1.4.12',
+    wcag1413: 'WCAG 1.4.13',
+    wcag211: 'WCAG 2.1.1',
+    wcag212: 'WCAG 2.1.2',
+    wcag214: 'WCAG 2.1.4',
+    wcag221: 'WCAG 2.2.1',
+    wcag222: 'WCAG 2.2.2',
+    wcag231: 'WCAG 2.3.1',
+    wcag241: 'WCAG 2.4.1',
+    wcag242: 'WCAG 2.4.2',
+    wcag243: 'WCAG 2.4.3',
+    wcag244: 'WCAG 2.4.4',
+    wcag245: 'WCAG 2.4.5',
+    wcag246: 'WCAG 2.4.6',
+    wcag247: 'WCAG 2.4.7',
+    wcag251: 'WCAG 2.5.1',
+    wcag252: 'WCAG 2.5.2',
+    wcag253: 'WCAG 2.5.3',
+    wcag254: 'WCAG 2.5.4',
+    wcag255: 'WCAG 2.5.5',
+    wcag311: 'WCAG 3.1.1',
+    wcag312: 'WCAG 3.1.2',
+    wcag321: 'WCAG 3.2.1',
+    wcag322: 'WCAG 3.2.2',
+    wcag323: 'WCAG 3.2.3',
+    wcag324: 'WCAG 3.2.4',
+    wcag331: 'WCAG 3.3.1',
+    wcag332: 'WCAG 3.3.2',
+    wcag333: 'WCAG 3.3.3',
+    wcag334: 'WCAG 3.3.4',
+    wcag411: 'WCAG 4.1.1',
+    wcag412: 'WCAG 4.1.2',
+    wcag413: 'WCAG 4.1.3',
+};
+
+export const getGuidelineKeyByAxeTag = (axeTag: string): string => {
+    return axeTagToGuidelineKeyMap[axeTag];
+};
+
+export const guidelineMetadata = {
+    'WCAG 1.1.1': {
+        number: '1.1.1',
+        axeTag: 'wcag111',
+        name: 'Non-text Content',
+        linkName: 'WCAG 1.1.1',
         linkTag: 'WCAG_1_1_1',
         link: 'https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html',
         guidanceTags: [],
     },
-    wcag121: {
-        name: 'WCAG 1.2.1',
+    'WCAG 1.2.1': {
+        number: '1.2.1',
+        axeTag: 'wcag121',
+        name: 'Audio-only and Video-only (Prerecorded)',
+        linkName: 'WCAG 1.2.1',
         linkTag: 'WCAG_1_2_1',
         link: 'https://www.w3.org/WAI/WCAG21/Understanding/audio-only-and-video-only-prerecorded',
         guidanceTags: [],
     },
-    wcag122: {
-        name: 'WCAG 1.2.2',
+    'WCAG 1.2.2': {
+        number: '1.2.2',
+        axeTag: 'wcag122',
+        name: 'Audio Description or Media Alternative (Prerecorded)',
+        linkName: 'WCAG 1.2.2',
         linkTag: 'WCAG_1_2_2',
         link: 'https://www.w3.org/WAI/WCAG21/Understanding/captions-prerecorded.html',
         guidanceTags: [],
     },
-    wcag123: {
-        name: 'WCAG 1.2.3',
+    'WCAG 1.2.3': {
+        number: '1.2.3',
+        axeTag: 'wcag123',
+        name: 'Audio description or media alternative (prerecorded)',
```

Comment:
[nit] All of the other names seem to match the capitalization of the linked title
```suggestion
        name: 'Audio Description or Media Alternative (Prerecorded)',
```